packages/clangml/clangml.3.5.0/files/clangml.install

```diff
@@ -0,0 +1,3 @@
+bin: [
+  "_build/consumer/processor.native" {"clangml-processor"}
+]
```

Comment:
It would be nice if this could be upstreamed into the package distribution, but not necessary.